components/script/dom/bluetooth.rs

```diff
@@ -443,6 +443,18 @@ impl BluetoothMethods for Bluetooth {
         return p;
     }
 
+    #[allow(unrooted_must_root)]
```

Comment:
@jdm That sounds very weird, why does `Rc<Promise>` need `unrooted_must_root` be allowed?